bad.py

```diff
@@ -0,0 +1,4 @@
+5 == 5
+
+concat_list = ["str1" "str2", "str3"]
+6 == 6
```

Comment:
## semgrep-action
**Severity:** Error :red_circle:
**Rule id:** python.lang.correctness.useless-eqeq.useless-eqeq
**Message:** This is always True: `6 == 6` or `6 != 6`. If testing for floating point NaN, use `math.isnan(6)`,
or `cmath.isnan(6)` if the number is complex.

<sub>[Report an issue with this rule.](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=bug_report.md&title=python.lang.correctness.useless-eqeq.useless-eqeq)</sub>

Reply:
<img src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDgiIGhlaWdodD0iNTMiIHZpZXdCb3g9IjAgMCA0OCA1MyIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0yOC40MzQgMS4xODQzNkMyNS42OTAzIC0wLjM5NDc4NSAyMi4zMDk4IC0wLjM5NDc4NiAxOS41NjYgMS4xODQzNkw0LjQzNDA1IDkuODkzMjRDMS42OTAyNSAxMS40NzI0IDAgMTQuMzkwOCAwIDE3LjU0OVYzNC45NjY4QzAgMzguMTI1MSAxLjY5MDI1IDQxLjA0MzUgNC40MzQwNSA0Mi42MjI2TDE5LjU2NiA1MS4zMzE1QzIyLjMwOTcgNTIuOTEwNiAyNS42OTAzIDUyLjkxMDYgMjguNDM0IDUxLjMzMTVMNDMuNTY1OSA0Mi42MjI2QzQ2LjMwOTcgNDEuMDQzNSA0OCAzOC4xMjUxIDQ4IDM0Ljk2NjhWMTcuNTQ5QzQ4IDE0LjM5MDggNDYuMzA5NyAxMS40NzI0IDQzLjU2NTkgOS44OTMyNEwyOC40MzQgMS4xODQzNlpNMzYuNzczOCAxNi4xNzYxTDI2LjI3MzcgMTAuMTIyMkMyNC44NjY3IDkuMzEwOTkgMjMuMTMzMiA5LjMxMDk5IDIxLjcyNjIgMTAuMTIyMkwxMS4yMjYxIDE2LjE3NjFDOS44MTkxIDE2Ljk4NzMgOC45NTIzNCAxOC40ODY1IDguOTUyMzQgMjAuMTA5VjMyLjIxNjdDOC45NTIzNCAzMy44MzkyIDkuODE5MSAzNS4zMzg0IDExLjIyNjEgMzYuMTQ5NkwyMS43MjYyIDQyLjIwMzVDMjMuMTMzMiA0My4wMTQ3IDI0Ljg2NjcgNDMuMDE0NyAyNi4yNzM3IDQyLjIwMzVMMzYuNzczOCAzNi4xNDk2QzM4LjE4MDggMzUuMzM4NCAzOS4wNDc2IDMzLjgzOTIgMzkuMDQ3NiAzMi4yMTY3VjIwLjEwOUMzOS4wNDc2IDE4LjQ4NjUgMzguMTgwOCAxNi45ODczIDM2Ljc3MzggMTYuMTc2MVoiIGZpbGw9IiNGNUY4RkEiLz4KPC9zdmc+Cg==" />

Reply:
<svg d="M11.4135 0.721676C13.014 -0.240559 14.986 -0.240558 16.5865 0.721676L25.4135 6.02835C27.014 6.99058 28 8.76886 28 10.6933V21.3067C28 23.2311 27.014 25.0094 25.4135 25.9717L16.5865 31.2783C14.986 32.2406 13.014 32.2406 11.4135 31.2783L2.58653 25.9717C0.985979 25.0094 0 23.2311 0 21.3067V10.6933C0 8.76886 0.98598 6.99058 2.58653 6.02835L11.4135 0.721676ZM15.2871 6.273C14.4906 5.79307 13.5093 5.79307 12.7129 6.273L6.62046 9.94413C5.82398 10.4241 5.33333 11.311 5.33333 12.2709V19.6132C5.33333 20.573 5.82398 21.46 6.62046 21.9399L12.7129 25.6111C13.5093 26.091 14.4906 26.091 15.2871 25.6111L21.3795 21.9399C22.176 21.46 22.6667 20.573 22.6667 19.6132V12.2709C22.6667 11.311 22.176 10.4241 21.3795 9.94413L15.2871 6.273Z" />

Reply:
<svg width="28" height="32" viewBox="0 0 28 32" fill="none" class="sc-AxirZ bJCmFu"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.4135 0.721676C13.014 -0.240559 14.986 -0.240558 16.5865 0.721676L25.4135 6.02835C27.014 6.99058 28 8.76886 28 10.6933V21.3067C28 23.2311 27.014 25.0094 25.4135 25.9717L16.5865 31.2783C14.986 32.2406 13.014 32.2406 11.4135 31.2783L2.58653 25.9717C0.985979 25.0094 0 23.2311 0 21.3067V10.6933C0 8.76886 0.98598 6.99058 2.58653 6.02835L11.4135 0.721676ZM15.2871 6.273C14.4906 5.79307 13.5093 5.79307 12.7129 6.273L6.62046 9.94413C5.82398 10.4241 5.33333 11.311 5.33333 12.2709V19.6132C5.33333 20.573 5.82398 21.46 6.62046 21.9399L12.7129 25.6111C13.5093 26.091 14.4906 26.091 15.2871 25.6111L21.3795 21.9399C22.176 21.46 22.6667 20.573 22.6667 19.6132V12.2709C22.6667 11.311 22.176 10.4241 21.3795 9.94413L15.2871 6.273Z"></path></svg>

Reply:
|table1|table|table3|